Installation/CMakeLists.txt

```diff
@@ -643,9 +640,8 @@ foreach (lib ${CGAL_SUPPORTING_3RD_PARTY_LIBRARIES})
   # add option
   list( FIND CGAL_ESSENTIAL_3RD_PARTY_LIBRARIES "${lib}" POSITION )
   if ( "${POSITION}" STRGREATER "-1" ) # if lib is essential
-   option(WITH_${lib} "Select external library ${lib}" ON)
-  else()
-    option(WITH_${lib} "Select external library ${lib}" OFF)
+
+  else() #GMPXX set in CGAL_SetupGMP.cmake
   endif()
```

Comment:
The patch does not look good. It has removed all the options `WITH_${lib}`. I agree that those options should eventually disappear, but I do not think that is the intent of this PR.

Reply:
Indeed, this is an error